### PR TITLE
U

### PR DESCRIPTION
**Pull request by Tmoneytroyt**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

IMG_0189

### CODE REVIEW

File: domains/troytech.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "description": "Troytech",
+  "repo": "https://github.com/Tmoneytroyt/register",
+  "owner": {
+    "username": "tmoneytroyt",
+    "email": "troysmithson12@icloud.com"
+  },
+  "record": {
+    "A": [
+      "75.2.60.5"
+    ]
```
Comment (CuteDog5695) [inline domains/troytech.json:11]: Please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/troytech.json (hunk 1 above)
Comment (Tmoneytroyt) [reply domains/troytech.json:11]: It’s about my blog site

Comment (Tmoneytroyt) [reply domains/troytech.json:11]: I attached it

**Comment by Tmoneytroyt** [comment]
Finished the website preview Ira a blog

**Pull request closed without merging**